cli/src/main/java/bisq/cli/app/BisqCliMain.java

```diff
@@ -41,60 +45,28 @@
 
     private final ManagedChannel channel;
     private final CliCommand cmd;
+    private final OptionParser cmdParser;
 
     public static void main(String[] args) {
-        new BisqCliMain("localhost", 8888);
+        new BisqCliMain("localhost", 8888, args);
```

Comment:
```suggestion
        new BisqCliMain("localhost", 9998, args);
```

8888 is a poor number as it's frequently used by other services. 9998 fits with bisq's p2p port of 9999 and follows suit with bitcoind's 8333 p2p port and 8332 rpc port. I realize you did not introduce this change and that it's somewhat out of scope for this PR, but please make it anyway.

@freimair, please note that the choice of 9998 here overlaps with the same port used in `monitor/src/main/java/bisq/monitor/metric/TorHiddenServiceStartupTime.java`. Please consider changing that to something else if you see it being a conflict.

Reply:
Changed in cli, but still need to decide how to resolve conflict with port number(s) in `monitor/src/main/java/bisq/monitor/metric/TorHiddenServiceStartupTime.java`

Reply:
TODO  change BisqGrpcServer's listening port.

Reply:
BisqGrpcServer now listens on port 9998.

Reply:
A reminder to @freimair, re `TorHiddenServiceStartupTime`
```
int localPort = Integer.parseInt(configuration.getProperty(LOCAL_PORT, "9998"));
int servicePort = Integer.parseInt(configuration.getProperty(SERVICE_PORT, "9999"));
```
now conflicts with the `BisqGrpcServer` listening port `9998` in this `remove-read-stdin-loop` branch (not merged).   I am sure I should not be changing server + listening ports in `TorHiddenServiceStartupTime`.

Reply:
I do not think so. the monitor is quite separate from everything else, plus, this is a default fallback value - it can and should be configured via the monitors properties file.